modules/mapper-extras/src/main/java/org/opensearch/index/mapper/RankFeaturesFieldMapper.java

```diff
@@ -84,16 +98,23 @@ public RankFeaturesFieldMapper build(BuilderContext context) {
 
     public static final class RankFeaturesFieldType extends MappedFieldType {
 
-        public RankFeaturesFieldType(String name, Map<String, String> meta) {
+        private final boolean positiveScoreImpact;
```

Comment:
It seems like `positiveScoreImpact` is not needed for `RankFeaturesFieldType`, the `positiveScoreImpact()` is never used